pages/faq/troubleshooting/faq.md

```diff
@@ -24,6 +24,7 @@ title: FAQs
 * [How do you secure your own “cloud” to prevent malicious attack which may allow attacker to break-in our systems?](#how-do-you-secure-your-own-cloud-to-prevent-malicious-attack-which-may-allow-attacker-to-break-in-our-systems)
 * [What does it mean when a device type is discontinued?](#what-does-it-mean-when-a-device-type-is-discontinued)
 * [I have a device that is not on the supported devices list. Can it run on {{ $names.company.lower }}?](#i-have-a-device-that-is-not-on-the-supported-devices-list-can-it-run-on-{{ $names.company.lower }})
+* [What to keep in mind while buying or using power supply units?](#what-to-keep-in-mind-while-buying-or-using-power-supply-units)
```

Comment:
Perhaps "while buying or using" could be shortened to just "when choosing" ?
And perhaps "Things to keep in mind" works better than "What to keep in mind" ? :shrug:

Reply:
:100: :raised_hands:

Reply:
I think I will go with the first suggestion only since we want to keep it still a question as this is a FAQ.